specification/glossary.md

```diff
@@ -84,6 +84,18 @@ A structured collection of cost and usage data that meets or exceeds the Basic c
 
 A Date/Time Format value that is contained within the beginning bound of a time period.
 
+<a name="glossary:instance-family"><b>Instance Family</b></a>
+
+A SKU property that groups compute-based SKUs with similar hardware and features that differ by allocated amounts of compute, memory, or storage.
+
+<a name="glossary:instance-size"><b>Instance Size</b></a>
+
+A SKU property denoting the SKU's relative computing capability within its [*instance family*](#glossary:instance-family).
+
+<a name="glossary:instance-type-ratio"><b>Instance Type Ratio</b></a>
```

Comment:
This seems to be miss-named.

Reply:
Also this is implying that the only supported approach to normalization factor is instance family and size. Normalization likely needs to be more generic to support multple normalization factors.

Reply:
I aggree with @ijurica this needs more consensus before introduction. This is especially true if we are going to use it to introduce a normalization factor.